geoutils/georaster.py

```diff
@@ -631,8 +631,8 @@ def clip(self):
 
     def reproject(self, dst_ref=None, dst_crs=None, dst_size=None,
                   dst_bounds=None, dst_res=None, nodata=None, dtype=None,
-                  resampling=Resampling.nearest, silent=False,
-                  **kwargs):
+                  resampling=Resampling.nearest, silent=False, n_threads=0,
+                  memory_limit=64):
```

Comment:
Should we leave it to default (0)?
At first I thought this was in GB, so I was a bit surprised!!

Reply:
If it's 0, an appropriate amount will be calculated. If >0, it will use that amount of threads.

Reply:
I meant for the memory_limit, not n_threads. If set to 0, it will use rasterio's default. But not a big deal...